src/map/clif.c

```diff
@@ -23308,17 +23308,17 @@ static void clif_rodex_read_mail(struct map_session_data *sd, int8 opentype, str
 
 	int fd = sd->fd;
 	int body_len = (int)strlen(msg->body) + 1;
-	int size = sizeof(struct PACKET_ZC_ACK_READ_RODEX);
+	int size = sizeof(struct PACKET_ZC_ACK_READ_MAIL);
```

Comment:
rename rodex into mail probably not good idea
because in herc rodex and mail packets based on different code. and because this should be not mixed.
technically in herc at same time can works old mail system and rodex.

Reply:
![grafik](https://user-images.githubusercontent.com/3517879/149022277-1497a4f5-b22d-4e2e-b648-f3728f03bb96.png)

No it cannot technically work at the same time in herc. :-)

Reply:
But feel free to adjust it to your likings. It was just a suggestion.
I have allowed edits by maintainers, so you can easily change it as well.

Reply:
i mean in herc mail and rodex is separate things, it may be not good, but it's like this for now.
this mean mail contains own packets for send/read etc and rodex contains own packets for do same. it more like duplication.
and while mail and rodex not merged, better name mail and rodex packets in different way.